src/effects/native/balanceeffect.cpp

```diff
@@ -55,13 +59,48 @@ EffectManifest BalanceEffect::getManifest() {
     midSide->setMaximum(1.0);
     midSide->setDefault(0.5);
 
+    EffectManifestParameter* midLowPass = manifest.addParameter();
+    midLowPass->setId("bypassFreq");
+    midLowPass->setName(QObject::tr("Bypass Freq."));
```

Comment:
This does not fit in the space for parameter names in Deere. How about "Bypass Frequency" for full name and just "Bypass" for short name?